src/rust/dbv2/migrations/2024-01-17-184459_coingecko/down.sql

```diff
@@ -0,0 +1,60 @@
+-- This file should undo anything in `up.sql`
+DROP VIEW api.historical_trades;
+DROP FUNCTION api.orderbook;
+DROP VIEW api.tickers;
+DROP FUNCTION api.get_market_liquidity;
+DROP FUNCTION api.get_market_24h_low;
+DROP FUNCTION api.get_market_24h_high;
+DROP FUNCTION api.get_market_bid;
+DROP FUNCTION api.get_market_ask;
```

Comment:
Per another comment I think this will need to be changed

Running the migrations checker after should then be in order